### ISSUE TITLE
Not getting any points for searches

### ISSUE THREAD

**Issue by jerkmcjerkface** [sealemar/BingRewards#300]
Just set this up, the environment is correct, but I'm not getting any points.  Looking at some other issues, it seems that the accounts need to be level 2 for the month before it works?

Running this on Macos with the latest versions:

BingRewards eric$ python -V
Python 2.7.10

Erics-Mac-mini:BingRewards eric$ sudo pip install beautifulsoup4
Requirement already satisfied: beautifulsoup4 in /Library/Python/2.7/site-packages

The output looks normal:

Live - account@gmail.com
Result 1/15:
-----------
name        : Browse & earn
url         : https://www.bing.com/explore/rewards-browse-and-earn?FORM=ML13VT&CREA=ML13VT&wt.mc_id=ML13VT&rnoreward=1
progressCur : 0
progressMax : 150
progress %  : 0.00%
description : Earn up to 150 points per month, 5 points per hour, for actively browsing with Microsoft Edge. Counter updates daily.
   Error    :   true
   Message  : Didn't earn any points for click
   Action   : warn

Result 2/15:
-----------
name        : Daily search
url         : https://www.bing.com/news?q=us+news&FORM=ML11Z9&CREA=ML11Z9&wt.mc_id=ML11Z9&rnoreward=1&rnoreward=1
progressCur : 15
progressMax : 50
progress %  : 30.00%
description : Earn up to 50 points per day, 5 points per search.
   Error    :   true
   Message  : Didn't earn any points for click
   Action   : warn

Live - account@gmail.com

Points before:      2566
Points after:       2566
Points earned:         0
Lifetime Credits: 113916

Am I missing something obvious?

**Comment by specu** [comment]
yes, account needs to be level 2. you can get to level 2 within a day.

**Comment by thatsmyface** [comment]
Also happening for me, my accounts are level 2

**Comment by zengfu94** [comment]
fix is here: https://github.com/zengfu94/MicrosoftRewards/commit/71d776f5304392d4fafe5e7d7999d4f1dd47ccf3

**Comment by imckumar** [comment]
Hi @zengfu94 ,
Same issue with me. not earning points or earning only 20 points:
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-= FINAL REPORT =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=

            Account           | Before | After  | Earned |  Lifetime  | Retries 
------------------------------+--------+--------+--------+------------+---------
 xxxxx@hotmail.com      |  10400 |  10420 |     20 |      16920 |       1
 xxxxx@hotmail.com    |  10990 |  11010 |     20 |      17510 |       1
Total points earned: 40

2017-11-11 12:19:55.3 - script ended
-------------------------
I downloaded latest script and tried running MicrosoftRewards and get below error:
root@ubuntu-trusty-1:~/MicrosoftRewards-master# ./bing.py 
Traceback (most recent call last):
  File "./bing.py", line 39, in <module>
    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
AttributeError: 'module' object has no attribute 'disable_warnings'

**Comment by zengfu94** [comment]
@imckumar your requests is out of date MicrosoftRewards works, run: pip install --upgrade  requests

**Comment by imckumar** [comment]
@zengfu94 
Can't make it to work. not sure what else is missing. Please check output https://pastebin.com/e45wHnmA

**Comment by zengfu94** [comment]
@imckumar can you join this chat on Telegram: https://t.me/joinchat/C30ZkkPAoCHXEuQCcJ5kvA I can help easier there

**Comment by Brenex** [comment]
@zengfu94 what happened to microsoftrewards??

**Comment by zengfu94** [comment]
@Brenex can't really say, hopefully u got last version before it went down

**Comment by fa14alonso** [comment]
For me searches are running but no points are being registered on the Rewards page. Started a few days ago

**Comment by thatsmyface** [comment]
Same issue here

**Comment by ethantrott** [comment]
any solution yet?

**Comment by imckumar** [comment]
Is it working for anyone ? 

For me, searches will work on some Random days and don't work for another couple of days and i receive only 20 points. 

Points before:       635
Points after:        655
Points earned:        20
Lifetime Credits:  20155
I already downloaded the latest version again.
